### PR TITLE
easyeffects: update to 7.1.7.

### PR DESCRIPTION
**Pull request by ACR-Jeff**
#### Testing the changes
- I tested the changes in this PR: briefly

#### Local build testing
- I built this PR locally for my native architecture, (x64)

This is my first commit, If I have done something incorrect please educate me lol. Thank you

### CODE REVIEW

File: srcpkgs/easyeffects/template [hunk 1]
```diff
@@ -10,12 +10,12 @@ makedepends="gtkmm4-devel pipewire-devel zita-convolver-devel lilv-devel
  soundtouch-devel speexdsp-devel json-c++ tbb-devel libadwaita-devel fmt-devel
  gsl-devel ladspa-sdk"
 short_desc="Sound effects for systems using PipeWire"
-maintainer="zenobit <zenobit@disroot.org>"
```
Comment (classabbyamp) [inline srcpkgs/easyeffects/template:13]: don't change this

Comment (ACR-Jeff) [reply srcpkgs/easyeffects/template:13]: Ok, Thank you, I should then create another branch and follow through correctly?

Comment (classabbyamp) [reply srcpkgs/easyeffects/template:13]: no, amend the commit and force-push

Comment (ACR-Jeff) [reply srcpkgs/easyeffects/template]: Thank you, Got it all figured out and pushed.

Comment (classabbyamp) [reply srcpkgs/easyeffects/template]: what i meant was:
```
git add srcpkgs/easyeffects/template
git commit --amend
git push --force
```
looks like you made a second commit instead. to fix that:
```
git reset --soft HEAD^
git commit --amend
git push --force
```

Comment (ACR-Jeff) [reply srcpkgs/easyeffects/template]: I am now lost, I thought I had it correct, Yet after following the instructions it pushed some other stuff in with it?

Comment (classabbyamp) [reply srcpkgs/easyeffects/template]: github showed two commits so I'm not sure either

**Pull request closed without merging**